Lib/_strptime.py

```diff
@@ -316,9 +317,9 @@ def _calc_julian_from_V(iso_year, iso_week, iso_weekday):
 
 
 def _strptime(data_string, format="%a %b %d %H:%M:%S %Y"):
-    """Return a 2-tuple consisting of a time struct and an int containing
+    """Return a 3-tuple consisting of a time struct and an int containing
     the number of microseconds based on the input string and the
-    format string."""
+    format string, and the GMT offset."""
```

Comment:
We should probably consistently use "UTC offset".

Reply:
@pganssle I am not really confident with the offsets and the datetime. Do you think we could keep it like that and propose an other bpo ?

Reply:
I just meant use UTC instead of GMT everywhere.